src/main/webapp/assets-lti/js/controllers/controllers.js

```diff
@@ -218,7 +218,7 @@ canvasSupportApp.controller('courseController', ['Course', 'Courses', 'Sections'
 
   $scope.getCoursesForTerm = function() {
     $scope.loadingOtherCourses = true;
-    var coursesUrl='/sectionsUtilityTool/manager/api/v1/courses?as_user_id=sis_login_id:' +uniqname+ '&per_page=200&published=true&with_enrollments=true&enrollment_type=teacher&_='+ generateCurrentTimestamp();
+    var coursesUrl='/canvasCourseManager/manager/api/v1/courses?as_user_id=sis_login_id:' +uniqname+ '&per_page=200&published=true&with_enrollments=true&enrollment_type=teacher&_='+ generateCurrentTimestamp();
```

Comment:
Since /canvasCourseManager/manager/api/v1 is the same everywhere could it be injected, or at least factored out into a variable?